web/frontend/src/components/slam/2d/index.svelte

```diff
@@ -0,0 +1,61 @@
+<script lang="ts">
+
+import { createEventDispatcher } from 'svelte';
+import { Canvas } from '@threlte/core';
+import * as THREE from 'three';
+import type { commonApi } from '@viamrobotics/sdk';
+
+import Legend from './legend.svelte';
+import Dropzone from '@/lib/components/dropzone.svelte';
+import Scene from './scene.svelte';
+
+export let pointcloud: Uint8Array | undefined;
+export let pose: commonApi.Pose | undefined;
+export let destination: THREE.Vector2 | undefined;
+export let helpers: boolean;
```

Comment:
Can you please leave docstrings explaining these props? e.g. how is pointcloud structured?

It would also be helpful to have file-level docstrings